### ISSUE TITLE
Error unzipping file with "backslashes as path separators"

### ISSUE THREAD

**Issue by timia2109** [mozilla/addons#290]
The Addon (on the tests on firefox it works and on the WebStore there was no error)
[ccsync.zip](https://github.com/mozilla/addons/files/613909/ccsync.zip)

I try to upload a new addon on the AMO (https://addons.mozilla.org/de/developers/addon/submit/upload-listed) and I only got the error:

> Sorry, we couldn't load your WebExtension.
> Fehler: Validation was unable to complete successfully due to an unexpected error.

So it's not listed or anything
I try to upload it at 25.11.2016 22:51 GTM+1

I found nothing what is wrong, beause on debug mode all run :(.

Hope you can help!
Thanks, Tim

**Comment by andymckay** [comment]
When unzipping you file locally I got:

`warning:  ccsync.zip appears to use backslashes as path separators`

Not sure if that's the cause, but I was able to reproduce the problem on a test AMO server. I rezipped the file using zip on OS X and uploaded it to a test AMO server and it got past that problem. It then reported a JavaScript error:

https://addons-dev.allizom.org/en-US/developers/upload/a449d3a357c54521a4df5585f3f8f76f

You'll need to fix that to continue, but also curious how you zipped it.
